dockerfiles/mesh-bridge/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Note: this must be built from the root of the project with:
+#
+#     docker build . -f ./cmd/mesh-bridge/Dockerfile
```

Comment:
```suggestion
#     docker build . -f ./dockerfiles/mesh-bridge/Dockerfile
```

Can you also add a command to the Makefile? I forgot that we did this for our other dockerfiles.